src/main/java/com/Project/accountManager/entities/User.java

```diff
@@ -0,0 +1,24 @@
+package com.Project.accountManager.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Table(name="user")
```

Comment:
@serhatbsts , Mysql contains some reserved keywords. that's why you can't use some keywords "user" "account" etc...
https://dev.mysql.com/doc/refman/8.0/en/keywords.html
![Screenshot 2024-05-24 at 06 18 49](https://github.com/serhatbsts/accountManager/assets/26492690/47390541-e30c-48f8-8a43-31c3781a2066)
![Screenshot 2024-05-24 at 06 19 06](https://github.com/serhatbsts/accountManager/assets/26492690/7cdac263-8dc8-404c-8743-a9400e47c7ee)

Reply:
OK, necessary corrections will be made.